### DISCUSSION TITLE
Unable to install rxode2

### DISCUSSION THREAD

**Discussion by faniekcheng** [nlmixr2/nlmixr2#82] [General]
I was trying to install nlmixr2 developer version but couldn't as cannot install rxode2. I managed to install an older version of rxode2 (2.0.7) but when I tried to download the developer's version, the folder was empty with no error messages.

I also tried the CRAN rxode2 version, it said 'ERROR: dependency ‘rxode2ll’ is not available for package ‘rxode2’'. After numerous attempts, I still cannot install rxode2ll.

I'm using R 4.2.2 and Rstudio 2022.07.02.   OS: Ubuntu 22.04.1 LTS

Has anyone experienced a similar issue?

Print screen for the developer version feedback below.
![image002](https://user-images.githubusercontent.com/101330424/202753252-d38fd23e-213f-46b3-8353-5ed5d007dda9.png)

**Comment by mattfidler** [comment]
My guess is the system dependencies are off. 

One option (the one I use) is to use a R package manager that installs the packages and the system dependencies.  For Ubuntu (one of the more commonly tested Linux distributions for R), you can use [r2u](https://github.com/eddelbuettel/r2u)

This seems to work just fine.  Then if you want to install the development version you can use the versions on `R-universe`
